easybuild/easyconfigs/p/polars/polars-1.28.1-gfbf-2023b.eb

```
@@ -0,0 +1,1079 @@
+    {'windows_x86_64_gnu-0.48.5.tar.gz': '53d40abd2583d23e4718fddf1ebec84dbff8381c07cae67ff7768bbf19c6718e'},
+    {'windows_x86_64_gnu-0.52.6.tar.gz': '147a5c80aabfbf0c7d901cb5895d1de30ef2907eb21fbbab29ca94c5b08b1a78'},
+    {'windows_x86_64_gnullvm-0.48.5.tar.gz': '0b7b52767868a23d5bab768e390dc5f5c55825b6d30b86c844ff2dc7414044cc'},
+    {'windows_x86_64_gnullvm-0.52.6.tar.gz': '24d5b23dc417412679681396f2b49f3de8c1473deb516bd34410872eff51ed0d'},
+    {'windows_x86_64_msvc-0.48.5.tar.gz': 'ed94fce61571a4006852b7389a063ab983c02eb1bb37b47f8272ce92d06d9538'},
+    {'windows_x86_64_msvc-0.52.6.tar.gz': '589f6da84c646204747d1270a2a5661ea66ed1cced2631d546fdfb155959f9ec'},
+    {'wit-bindgen-rt-0.33.0.tar.gz': '3268f3d866458b787f390cf61f4bbb563b922d091359f9608842999eaee3943c'},
+    {'write16-1.0.0.tar.gz': 'd1890f4022759daae28ed4fe62859b1236caebfc61ede2f63ed4e695f3f6d936'},
+    {'writeable-0.5.5.tar.gz': '1e9df38ee2d2c3c5948ea468a8406ff0db0b29ae1ffde1bcf20ef305bcc95c51'},
+    {'x11rb-0.13.1.tar.gz': '5d91ffca73ee7f68ce055750bf9f6eca0780b8c85eff9bc046a3b0da41755e12'},
+    {'x11rb-protocol-0.13.1.tar.gz': 'ec107c4503ea0b4a98ef47356329af139c0a4f7750e621cf2973cd3385ebcb3d'},
+    {'xmlparser-0.13.6.tar.gz': '66fee0b777b0f5ac1c69bb06d361268faafa61cd4682ae064a171c16c433e9e4'},
+    {'xxhash-rust-0.8.15.tar.gz': 'fdd20c5420375476fbd4394763288da7eb0cc0b8c11deed431a91562af7335d3'},
+    {'yoke-0.7.5.tar.gz': '120e6aef9aa629e3d4f52dc8cc43a015c7724194c97dfaf45180d2daf2b77f40'},
+    {'yoke-derive-0.7.5.tar.gz': '2380878cad4ac9aac1e2435f3eb4020e8374b5f13c296cb75b4620ff8e229154'},
+    {'zerocopy-0.7.35.tar.gz': '1b9b4fd18abc82b8136838da5d50bae7bdea537c574d8dc1a34ed098d6c166f0'},
+    {'zerocopy-derive-0.7.35.tar.gz': 'fa4f8080344d4671fb4e831a13ad1e68092748387dfc4f55e356242fae12ce3e'},
+    {'zerofrom-0.1.6.tar.gz': '50cc42e0333e05660c3587f3bf9d0478688e15d870fab3346451ce7f8c9fbea5'},
+    {'zerofrom-derive-0.1.6.tar.gz': 'd71e5d6e06ab090c67b5e44993ec16b72dcbaabc526db883a360057678b48502'},
+    {'zeroize-1.8.1.tar.gz': 'ced3678a2879b30306d323f4542626697a464a97c0a07c9aebf7ebca65cd4dde'},
+    {'zerovec-0.10.4.tar.gz': 'aa2b893d79df23bfb12d5461018d408ea19dfafe76c2c7ef6d4eba614f8ff079'},
+    {'zerovec-derive-0.10.3.tar.gz': '6eafa6dfb17584ea3e2bd6e76e0cc15ad7af12b09abdd1ca55961bed9b1063c6'},
+    {'zlib-rs-0.4.2.tar.gz': '8b20717f0917c908dc63de2e44e97f1e6b126ca58d0e391cee86d504eb8fbd05'},
+    {'zstd-0.13.3.tar.gz': 'e91ee311a569c327171651566e07972200e76fcfe2242a4fa446149a3881c08a'},
+    {'zstd-safe-7.2.3.tar.gz': 'f3051792fbdc2e1e143244dc28c60f73d8470e93f3f9cbd0ead44da5ed802722'},
+    {'zstd-sys-2.0.14+zstd.1.5.7.tar.gz': '8fb060d4926e4ac3a3ad15d864e99ceb5f343c6b34f5bd6d81ae6ed417311be5'},
+    {'polars-1.28.1_polars-utils.patch': '6dcd1d849a48855935e1b4303bf6fb86864bd4010dbeb58b4dfe9bff2751fc34'},
+]
+
+builddependencies = [('maturin', '1.8.3', '-Rust-1.86.0')]
```

@PetrKralCZ This will need to change, since we're using Rust as build-only dependency for maturin now, see #22981

```suggestion
builddependencies = [
    ('Rust', '1.86.0'),
    ('maturin', '1.6.0'),
]